SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3011,10 +3011,23 @@ void USpatialNetDriver::TryFinishStartup()
 				ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(GetWorld()->GetWorldSettings());
 				if (WorldSettings && WorldSettings->bEnableDebugInterface)
 				{
+					auto DebugCompFilter = [this](const Worker_EntityId EntityId, const SpatialGDK::EntityViewElement& Element) {
```

Comment:
do you need the filter? You can only see the debug tag on entities that have the debug component as it's behind a debug component constrained query, so this seems defensive

Reply:
ah, if you remove the component locally (and assuming a refresh) it would be inconsistent until the Runtime catches up and tells you you're no longer interested in the tag. Not sure that's a problem here though.